components/atoms/figure-component.html

```diff
@@ -0,0 +1,59 @@
+<div class="component" title="figure Component" version="1.0">
+<style>
+.figure {
+    display: grid;
+    margin: 0 auto;
+}
+
+.figure img {
+    width: 100%;
+    margin-bottom: 1em;
+}
+
+.figure figcaption {
+    text-align: center;
+}
+.figure figcaption a{
+    text-decoration: none;
+}
+</style>
+
+
+  <figure class="figure">
+      <img src="../../img/mustafa.jpg" alt="">
+    <figcaption>
+        <span>Mustafa Alsaeedi</span>
```

Comment:
The component shoudl have placeholder content, not real content. So instead of `Mustafa Alsaeedi` it should say `Lorem Ipsum`